seed/README.md

```diff
@@ -0,0 +1,61 @@
+# Challenges
+
+The `seed` directory contains all the challenges that appear on the FreeCodeCamp site. 
```

Comment:
FreeCodeCamp `=>` freeCodeCamp

Reply:
also, prefer:

`...challenges that appear on the freeCodeCamp learning platform.`